apps/sch/fsw/unit_test/sch_app_test.c

```diff
@@ -2253,9 +2253,20 @@ void SCH_ValidateMessageData_Test_MessageIdUnusedGarbageEntry(void)
     Result = SCH_ValidateMessageData(&SCH_AppData.MessageTable[0]);
     
     /* Verify results */
-    UtAssert_True
-        (Ut_CFE_EVS_EventSent(SCH_MESSAGE_TBL_ERR_EID, CFE_EVS_ERROR, "Message tbl verify err - idx[0] mid[0x1897] len[10] buf[0]"),
-        "Message tbl verify err - idx[0] mid[0x1897] len[10] buf[0]");
+    if (SCH_UNUSED_MID == 0x0)
```

Comment:
See my comment on the ci peer review regarding this particular change.

Reply:
Done and pushed